apps/autorelayer-interop/src/app.ts

```diff
@@ -95,6 +102,24 @@ class RelayerApp extends App {
       throw new Error(`failed to fetch chains: ${error}`)
     }
 
+    if (config.chainEndpointOverrides) {
+      for (const url of config.chainEndpointOverrides) {
+        try {
+          const client = createPublicClient({ transport: http(url) })
+          const chainId = await client.getChainId()
+          const chain = chains.find((c) => c.id === chainId)
+          if (!chain) {
+            continue
+          }
+          chain.url = url
```

Comment:
```suggestion
          if (chain) chain.url = url 
```

could make this 1 line